bin.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+console.error('`pino` cli is deprecated. Use `pino-pretty` cli instead.')
```

Comment:
Doesn't **deprecated** mean that something is still working but it's discouraged to use?
I'd change the message in here to `'pino' cli got removed. Use 'pino-pretty' cli instead.`

A link to https://www.npmjs.com/package/pino-pretty could also be useful.